Transport/Simple/F90/actual_transport_nonideal.f90

```diff
@@ -30,8 +30,8 @@ module actual_transport_module
   real(amrex_real), save, allocatable :: Afac(:,:)
   real(amrex_real), save, allocatable :: Bfac(:,:)
   real(amrex_real), save, allocatable :: Sigmaij(:,:)
-  real(amrex_real), parameter :: Avna = 6.022140857e23
-  real(amrex_real), parameter :: PI = 4 * atan (1.0)
+  real(amrex_real), parameter :: Avna = 6.022140857d23
```

Comment:
This is fine here, but should probably be redone across all the code to properly respect double/float compilation flags.